etcdserver/api/membership/cluster.go

```diff
@@ -252,6 +252,16 @@ func (c *RaftCluster) Recover(onSet func(*zap.Logger, *semver.Version)) {
 	}
 }
 
+// IsPromoteChange checks if m is a promoteChange
+func (c *RaftCluster) IsPromoteChange(m *Member) bool {
```

Comment:
IsMemberPromoted

Reply:
This function checks if the config change is for promoting a member (as opposed to adding a member). Maybe use `IsMemberPromoteConfChange` or `IsConfChangeForMemberPromote`?